ETL_Mini_Project_TRadadiya_ZLiang_YWong_FGirnary.ipynb

```diff
@@ -555,21 +703,24 @@
    ],
    "source": [
     "# Use a list comprehension to add \"cat\" to each category_id. \n",
+    "cat_ids = [\"cat\" + str(cat_id) for cat_id in category_ids]\n",
     "\n",
     "# Use a list comprehension to add \"subcat\" to each subcategory_id.    \n",
-    "\n",
+    "scat_ids = [\"subcat\" + str(subcat_id) for subcat_id in subcategory_ids]\n",
     "    \n",
     "print(cat_ids)\n",
     "print(scat_ids)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 18,
    "metadata": {},
    "outputs": [],
    "source": [
     "# Create a category DataFrame with the category_id array as the category_id and categories list as the category name.\n",
+    "category_df = pd.DataFrame({'category_id': cat_ids,\n",
+    "                            'category': categories})\n",
     "\n",
     "# Create a category DataFrame with the subcategory_id array as the subcategory_id and subcategories list as the subcategory name. \n"
    ]
```

Comment:
Add a code to create subcategory dataframe as below
subcategory_df = pd.DataFrame({'subcategory_id': scat_ids,
                            'subcategory': subcategories})